src/PresentationalComponents/Snippets/PackagesErrataLink.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Text, TextContent, TextVariants } from '@patternfly/react-core';
+import Label from './Label';
+import { handlePatchLink, truncate } from '../../Utilities/Helpers';
+import { entityTypes } from '../../Utilities/constants';
+import ExternalLink from './ExternalLink';
+import propTypes from 'prop-types';
+
+export const PackagesErrataLink = ({ row }) =>  (<TextContent>
+    <Label>Description</Label>
+    <Text component={TextVariants.p} style={{ whiteSpace: 'pre-line' }}>
+        {truncate(row.attributes.description.replace(
+            new RegExp('\\n(?=[^\\n])', 'g'),
+            ''
+        ), 570, handlePatchLink(entityTypes.advisories, row.id, 'Read more'))}
+    </Text>
+    <ExternalLink link={`https://access.redhat.com/errata/${row.id}`}
+        text={'View packages and errata at access.redhat.com'} />
+</TextContent>);
+
+PackagesErrataLink.propTypes = {
+    row: propTypes.object
```

Comment:
As we pass "row", can you please specify more detailed propType? That it should contain "id" and "attributes"

Reply:
Or mabe use directly props -> id and description. Also, the name of the component does not say that it contains also description. Maybe "DescriptionWithLink"? Or something like that